code/modules/surgery/organs/subtypes/standard.dm

```diff
@@ -13,7 +13,6 @@
 	vital = 1
 	amputation_point = "spine"
 	gendered_icon = 1
-	cannot_amputate = 1
```

Comment:
Don't forget groins.

Reply:
[This line](https://github.com/Aurorablade/Paradise/blob/acc2547e38289c22be286ca05d3022d63d4a384a/code/modules/surgery/organs/subtypes/standard.dm#L48 )